interview/infra/ocr/tesseract_client.py

```
@@ -0,0 +1,34 @@
+import pytesseract
+from pdf2image import convert_from_path
+from PIL import Image
+import os
+import tempfile
+
+
+class TesseractOCRClient:
+    def __init__(self, lang: str = "kor+eng"):
+        self.lang = lang
+
+    def extract_text_from_image(self, image_path: str) -> str:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)